test/language/module-code/export-expname-binding-string.js

```diff
@@ -0,0 +1,17 @@
+// Copyright (C) 2020 Bradley Farias. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+description: Module Namespace Objects can have non-Identifier properties
+esid: prod-ExportSpecifier
+info:
```

Comment:
The lint task is failing because without this, `info` is being interpreted as an object, rather than as a literal multiline string:

```suggestion
info: |
```